docs/orchestrator/concept.md

```diff
@@ -0,0 +1,119 @@
+## Turning tools into pipelines
```

Comment:
Shouldn't we stick to `workflow`?

Reply:
Not necessarily, as pipelines are something that is better understandable.

Reply:
I used "Combining tools into workflows"